docs/ldml/tr35.md

```diff
@@ -3803,13 +3803,12 @@ A matching rule can be used to transform the source fields as follows
 
 _Example:_
 
-> source=ja-Latn-fonipa-hepburn-heploc
+> source="ja-Latn-fonipa-hepburn-heploc"
 >
-> rule  ="\<languageAlias type="und-hepburn-heploc"
+> rule  =`\<languageAlias type="und-hepburn-heploc" replacement="und-alalc97">`
 >
-> replacement="und-alalc97">"
->
-> result="ja-Latn-alalc97-fonipa" // note that CLDR canonical order of variants is alphabetical
+> result="ja-Latn-alalc97-fonipa"
```

Comment:
I thought source and result were also supposed to be enclosed?

Reply:
I just changed the items that are fragments of XML syntax; that is more consistent with the rest of the document. Adding a note to the ticket.